src/docs/sphinx/quickstart.rst

```diff
@@ -267,28 +267,268 @@ must be specified using either:
+* By default, MacPorts installs packages in ``/opt/local``; the above paths need to be adjusted if you choose a different location.
+  This of course applies to the packages in ``packages.yaml`` as well.
+* As noted above, the ``port select ...`` commands will set which version of clang gets invoked by the executables ``/opt/local/bin/clang``, etc.
+  The paths above are thus valid only if you activated the ``clang`` package that matches the compiler spec.
+  Alternatively, you could set the full name and path of the executables of the desired compilers if you don't want MacPorts settings to interfere with your spack settings.
+* You should set the spec to be equal to the actual version of the compiler you installed.
+
+In ``packages.yaml``:
+
+.. code-block:: yaml
+
+   packages:
+   all:
+      compiler: [clang, gcc]
+      providers:
+         blas: [netlib-lapack]
+         lapack: [netlib-lapack]
+         mpi: [openmpi]
+
+   mpi:
+      buildable: false
+   openmpi:
+      externals:
+      - spec: openmpi@4.1.4
+         prefix: /opt/local
+   
+   netlib-lapack:
+      buildable: false
+      externals:
+      - spec: netlib-lapack@3.10.1
+         prefix: /opt/local
+   autoconf:
+      buildable: false
+      externals:
+      - spec: autoconf@2.71
+         prefix: /opt/local
+   automake:
+      buildable: false
+      externals:
+      - spec: automake@1.16.5
+         prefix: /opt/local
+   bzip2:
+      buildable: false
+      externals:
+      - spec: bzip2@1.0.8
+         prefix: /opt/local
+   gettext:
+      buildable: false
+      externals:
+      - spec: gettext@0.21
+         prefix: /opt/local
+   graphviz:
+      buildable: false
+      externals:
+      - spec: graphviz@2.50.0
+         prefix: /opt/local
+   libtool:
+      buildable: false
+      externals:
+      - spec: libtool@2.4.6
+         prefix: /opt/local
+   libx11:
+      buildable: false
+      externals:
+      - spec: libx11@1.8.1
+         prefix: /opt/local
+   m4:
+      buildable: false
+      externals:
+      - spec: m4@1.4.6
+         prefix: /usr
+   perl:
+      buildable: false
+      externals:
+      - spec: perl@v5.34.1
+         prefix: /opt/local
+   pkg-config:
+      buildable: false
+      externals:
+      - spec: pkg-config@0.29.2
+         prefix: /opt/local
+   tar:
+      buildable: false
+      externals:
+      - spec: tar@3.3.2
+         prefix: /usr
+   readline:
+      buildable: false
+      externals:
+      - spec: readline@8.1.2.000
+         prefix: /opt/local
+   unzip:
+      buildable: false
+      externals:
+      - spec: unzip@6.0
+         prefix: /usr
+   zlib:
+      buildable: false
+      externals:
+      - spec: zlib@1.2.12
+         prefix: /opt/local
+
+Notes:
+
+* OpenGL is not supported on modern Macs, hence those packages are not specified as they are on other platforms. 
```

Comment:
Depends on what you mean by "supported". macOS still supports them, but Spack doesn't use the correct locations when it attempts to look up headers and libraries in the `opengl` package on macOS 12.5 (and possibly earlier versions).

OpenGL has been deprecated since macOS 10.14, but the XCode Command Line Tools still install OpenGL 4.1 and GLU 1.3. On earlier versions of macOS, it used to be possible to specify the `opengl` package in `packages.yaml` with an install prefix. On newer versions of macOS (certainly macOS 12.5 on an M1 Mac), those paths have changed. The paths in the Spack `opengl` package could be updated using the path information in https://github.com/ziglang/zig/issues/2208#issuecomment-582040352, but it's probably not worth the hassle unless someone really wants to use `glvis` on macOS.